inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -655,16 +687,25 @@ year age group.
 ```{r cluster_hh_size}
 # get counts of the number of households per cluster
 clustersize <- study_data_cleaned %>% 
-  count(cluster)
+  group_by(cluster) %>% 
+  # return the number of unique household IDs by each cluster
+  summarise(
```

Comment:
This form of exploding nested calls is a bit distracting. it's not difficult to read
`summarise(n = length(unique(q14_hh_no)))`

Reply:
okie dokie - just thought it was neater to seperate out when theres more than one thing going on. but am okay eitherway!